Doc for Quable (#3085)
ezsystems/developer-documentation: docs/product_catalog/product_catalog_guide.md

Reviewer: ```suggestion
![How does product catalog work](img/how_pim_works.png)
```

---
description: The product catalog guide provides a full description of the features and capabilities for managing products, their specifications, variants, pricing, and organization.
month_change: false
---

# Product catalog guide

## What is product catalog

The product catalog is a comprehensive set of capabilities for managing products in [[= product_name =]] that can be used standalone.
It lets you create, configure, and manage products, their specifications, assets, variants, and prices, and group products into categories and catalogs.

## Availability

> Product Catalog capabilities are available in all [[= product_name =]] editions.

Reviewer: ```suggestion
Product catalog capabilities are available in all [[= product_name =]] editions.
```

## How does product catalog work

> Products in [[= product_name =]]’s PIM have underlying content items enriched with product-specific information such as attributes, assets, prices, and others.

Reviewer: ```suggestion
Products in [[= product_name =]]’s product catalog have underlying content items enriched with product-specific information such as attributes, assets, prices, and others.
```

> The Product Catalog lets you group products into categories and catalogs.

Reviewer: ```suggestion
The product catalog lets you group products into categories and catalogs.
```

- product category
- the date when the product was created

Catalog filters let you narrow down the products from the product catalog that are available in the given catalog.
Besides, the built-in catalog filters, you can also [create custom ones](create_custom_catalog_filter.md).

### Remote PIM support

[[= product_name =]] provides flexible product catalog infrastructure that works with external PIM systems.

In [[= product_name =]], products are created and maintained by using the REST API or the back office, and their data is stored in a local database.
However, in your project or organization, you might have an existing product database, or be specifically concerned about product information security.
To address such needs, [[= product_name =]] provides remote PIM support.

> You can use the existing [[[= pim_product_name =]] PIM integration](/product_catalog/quable/quable.md), or build a custom one to connect to a remote PIM or ERP system, pull product data and present it on your website.

Reviewer: ```suggestion
You can install and configure a readily available [[[= pim_product_name =]] PIM integration](/product_catalog/quable/quable.md) add-on, or build a custom one to connect to a remote PIM or ERP system, pull product data and present it on your website.
```

By default, such information is held in the [[= product_name =]]'s local database.
In your specific scenario, you can implement the support for availability and pricing information coming from an external source as well, by using a price/availability matching strategy that is an extension point exposed in the Product catalog module.

#### Limitations

The limitation of remote PIM depend on implementation details of specific integration and may arise in areas relying on [content model](content_model.md).

> To see limitations of the [[= pim_product_name =]] PIM integration, see [[[= pim_product_name =]] PIM known limitations](/product_catalog/quable/quable_guide.md#known-limitations).

Reviewer: ```suggestion
To see the limitations of the [[= pim_product_name =]] PIM integration add-on, see [[[= pim_product_name =]] PIM known limitations](/product_catalog/quable/quable_guide.md#known-limitations).
```
